src/plugins/LADTable/components/LADTable.vue

```diff
@@ -93,31 +105,61 @@ export default {
         this.composition.off('add', this.addItem);
         this.composition.off('remove', this.removeItem);
         this.composition.off('reorder', this.reorder);
+
+        for (const id of Object.keys(this.unsubsubscribes)) {
+            this.unsubsubscribes[id]();
```

Comment:
Was the double 'sub' intentional in `unsubsubscribes`?

Reply:
hahaha... whoops. I'll take care of that.